tests/puppeteer/pages/BO/BObasePage.js

```diff
@@ -76,6 +78,7 @@ module.exports = class BOBasePage extends CommonPage {
     // Alert Text
     this.alertSuccessBloc = 'div.alert.alert-success:not([style=\'display: none;\'])';
     this.alertSuccessBlockParagraph = `${this.alertSuccessBloc} div.alert-text p`;
+    this.alertTextBlock = '.alert-text';
```

Comment:
it's some thing different than `alertSuccessBlockParagraph` ?

Reply:
Yes it's different, it's not the success alert, I use it for the error message